app/api/v_sphere/connection.rb

```diff
@@ -37,7 +37,7 @@ def clusters_folder
     end
 
     def configured?
-      !@server_ip&.empty? && !@server_user&.empty? && !@server_password&.empty?
+      !@server_ip.nil? && !@server_user.nil? && !@server_password.nil?
```

Comment:
&.empty? was actually intended here, because that would catch both a nil-Configuration and an empty one.
Whats your reason for this change?

Reply:
I think that's the point @BraunTom sees...
![image](https://user-images.githubusercontent.com/24682367/53085459-36ad1e80-3503-11e9-82dd-5c03f278436d.png)

Reply:
When the value is nil, it evaluates to true.

Reply:
Ah, right...
Ok, I need to change that, but we should still check that the strings are not empty, as an empty configuration cannot be valid anyway

Reply:
Changed it so it checks for nil and empty string